### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by Harbinzzy** [blastnet/blastnet.github.io#25]
Hello, I downloaded the dataset through kaggle datasets download-d waitongchung/blastnet-momentum-3d-sr-dataset, but only HR/ and LR_16x/ were downloaded after downloading. There are no LR_8x/ or LR_32x/, and the.csv file only has all_data_summary.csv

**Comment by waitong94** [comment]
hi harbinzzy

i believe this is an issue from the kaggle web ui

can you try downloading with the web-cli? 

Tutorial link: https://blastnet.github.io/tutorial#kaggle-command-line-api

**Comment by waitong94** [comment]
i see that you're seeing this issue with the kaggle cli as well. let me try to replicate this

**Comment by Harbinzzy** [comment]
Ok, thank you very much for your help, I will download it from the web ui. But it's still only 54.1GB, the same size as the zip I downloaded via the kaggle cli.

**Comment by Harbinzzy** [comment]
Thank you very much for your help, I will try your suggestion




🙃
***@***.***

**Comment by Harbinzzy** [comment]
![image](https://github.com/user-attachments/assets/8f420f24-6384-4555-919b-af4c01bf2449)
![image](https://github.com/user-attachments/assets/4a093164-fbd0-4f3a-85cd-efaf77a42d88)
Thank you very much for your help

**Comment by Harbinzzy** [comment]
> i see that you're seeing this issue with the kaggle cli as well. let me try to replicate this.
> 
> in the meantime, can you try downloading from clicking the download link from the web ui?
only HR/ and LR_16x/ were downloaded after downloading. There are no LR_8x/ or LR_32x/, and the.csv file only has all_data_summary.csv from clicking the download link from the web ui

**Comment by Harbinzzy** [comment]
both kaggle cli and the web ui lack LR_8x/ or LR_32x/, and zip is only 54.1G. How do I fix this.

**Comment by pzhanggit** [comment]
Hi @Harbinzzy @waitong94, we're running into the same issue. It seems the SR datasets are incomplete, only 53~54GB as
```
.
├── best_model_weights
│   ├── seed42
│   ├── seed43
│   └── seed44
└── dataset
    ├── HR
    │   ├── forcedhit
    │   ├── paramvar
    │   ├── test
    │   ├── train
    │   └── val
    └── LR_16x
        ├── forcedhit
        ├── paramvar
        ├── test
        └── train
```

And there are only density files in `/dataset/LR_16x/train/`. Could you help us get the entire datasets? Thanks!

**Comment by waitong94** [comment]
Yeah I've found this be an issue on the Kaggle side. Can you use their web ui and click on the folders in the data tree and click download. 

I find that this helps with recovering missing data.

![Image](https://github.com/user-attachments/assets/3e4bbcb0-359e-45be-b6ac-4eae18ccf8ac)

**Comment by pzhanggit** [comment]
> Yeah I've found this be an issue on the Kaggle side. Can you use their web ui and click on the folders in the data tree and click download.
> 
> I find that this helps with recovering missing data.
> 
> ![Image](https://github.com/user-attachments/assets/3e4bbcb0-359e-45be-b6ac-4eae18ccf8ac)

Awesome, it worked! Thank you very much, @waitong94

**Comment by akoushb** [comment]
@Harbinzzy @pzhanggit  Thanks for reporting this issue! 

Every dataset on Kaggle has two reported sizes: compressed (ZIP) vs. uncompressed, and the difference typically ranges between 10-40% due to compression efficiency. Based on your reporting, It seems likely that your extraction process was interrupted before completion, possibly due to memory limitations.

`kaggle datasets download waitongchung/blastnet-momentum-3d-sr-dataset # shows 54.1 GB in download progress`
Once the dataset is downloaded and unzipped 
`unzip blastnet-momentum-3d-sr-dataset.zip-d blastnet-momentum-3d-sr-dataset`
`du -sh blastnet-momentum-3d-sr-dataset # 71 GB`

since unzipping ~32k files will take longer time, you can easily verify the dataset.zip before extracting and list all files without extracting them:
`unzip -l blastnet-momentum-3d-sr-dataset.zip > out.log`
This will generate out.log, which contains a list of all files inside the ZIP.

You should get the same result if you used the following:
`kaggle datasets download waitongchung/blastnet-momentum-3d-sr-dataset --unzip -p output`

If you followed these steps, you should have all the files from momentum-dataset ~ 32k files. If you’re still facing any issue with Kaggle CLI on BLASTNet Datasets, please let us know the following:
1. Kaggle version
2. OS
3. your CLI to download the dataset.
